src/main/java/frc/robot/subsystems/ClimbPivot.java

```diff
@@ -207,6 +208,9 @@ public void periodic() {
     } else {
       isClimbIn = false;
     }
+    if (nextMoveOut = true) {
+      holdPosition();
+    }
```

Comment:
This should immediately override the power set in `toggleClimb()`. Though I think this just creates a Command instance and then discards it, so actually has no effect.